presto-tests/src/test/java/io/prestosql/tests/TestInformationSchemaConnector.java

```diff
@@ -52,12 +52,12 @@
     public void testBasic()
     {
         assertQuery("SELECT count(*) FROM tpch.information_schema.schemata", "VALUES 10");
-        assertQuery("SELECT count(*) FROM tpch.information_schema.tables", "VALUES 80");
-        assertQuery("SELECT count(*) FROM tpch.information_schema.columns", "VALUES 583");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.tables", "VALUES 81");
```

Comment:
why this numbers got higher?

Reply:
This is because these include the new tables and columns in `information_schema` so both tables and columns have more values.